lottie/src/main/java/com/airbnb/lottie/animation/content/FillContent.java

```diff
@@ -98,9 +98,9 @@ public FillContent(final LottieDrawable lottieDrawable, BaseLayer layer, ShapeFi
       return;
     }
     L.beginSection("FillContent#draw");
-    paint.setColor(((ColorKeyframeAnimation) colorAnimation).getIntValue());
+    int color = ((ColorKeyframeAnimation) this.colorAnimation).getIntValue();
     int alpha = (int) ((parentAlpha / 255f * opacityAnimation.getValue() / 100f) * 255);
-    paint.setAlpha(clamp(alpha, 0, 255));
+    paint.setColor((clamp(alpha, 0, 255) << 24) | (color & 0xFFFFFF));
```

Comment:
@pablorengo You can revert this now right?

Reply:
@gpeal In this case, it avoids calling `setColor()` twice.

Reply:
Ah, I see. Got it. We could also make `setColorAndAlpha` on `LPaint` in case there are other usages of this in the future